### ISSUE TITLE
DatePicker does not layout nicely

### ISSUE THREAD

**Issue by MarcSkovMadsen** [holoviz/panel#922]
#### System Info

- Panel: 0.7.0
- Bokeh: 1.4.0
- Tornado 5.1.11
- Python: 3.7.4
- OS: Windows 8.1
- Browser: Chrome

#### Description of expected behavior and the observed behavior

I would expect the `pn.widgets.DatePicker` to layout nicely. But it does not.

![image](https://user-images.githubusercontent.com/42288570/71775743-4b14d100-2f86-11ea-98cf-adacba4b7ed3.png)

#### Complete, minimal, self-contained example code that reproduces the issue

```python
"""The DatePicker widget does not layout nicely"""

import panel as pn

app = pn.Column(__doc__, pn.widgets.DatePicker())
app.servable()
```

**Comment by philippjfr** [comment]
Hmm, very odd. If we can reproduce with just bokeh we should file it there.

**Comment by philippjfr** [comment]
That said, the DatePicker is being rewritten entirely right now so it may just go away in bokeh 2.0.

**Comment by philippjfr** [comment]
Appears to indeed be fixed in bokeh 2.0
